README.md

```diff
@@ -2,6 +2,14 @@
 
 [![Documentation Status](https://readthedocs.org/projects/microsoft-recommenders/badge/?version=latest)](https://microsoft-recommenders.readthedocs.io/en/latest/?badge=latest)
 
+## What's New
+---
+[Microsoft MIND Competition!](https://msnews.github.io/competition.html)
+
+Microsoft is hosting a News Recommendation competition based on the [MIND dataset](https://msnews.github.io/), a large-scale English news dataset with impression logs. Check out the [ACL paper](https://msnews.github.io/assets/doc/ACL2020_MIND.pdf), get familiar with the [news recommendation scenario](https://github.com/microsoft/recommenders/tree/master/scenarios/news), and dive into the [quick start example](examples/00_quick_start/dkn_MIND_dataset.ipynb) using the DKN algorithm. Then try some other algorthms and tools in recommenders and submit your entry!
+
+---
+
```

Comment:
the transition between what's new and the explanation of the repo is not clear, maybe we could add a new title section like `Introduction` or `Description` or something like that

Reply:
yep, it should look better now